test/image/mocks/annotations.json

```diff
@@ -17,6 +17,7 @@
             {"text":"right bottom","showarrow":false,"xref":"paper","yref":"paper","xanchor":"right","yanchor":"bottom","x":0.5,"y":1},
             {"text":"move with page","xref":"paper","yref":"paper","x":0.75,"y":1},
             {"text":"opacity","opacity":0.5,"x":5,"y":5},
+            {"text":"not-visible", "visible": false},
```

Comment:
(actually that's true of https://github.com/plotly/plotly.js/commit/a2dd6347f0ba9b4fd8b8741a6e61dc06642dff17 too right?)